### ISSUE TITLE
Docker Daemon Config tests on CoreOS when using early-docker

### ISSUE THREAD

**Issue by dhawal55** [docker/docker-bench-security#152]
I'm running Kubernetes on CoreOS and using early-docker to start flannel (network overlay for k8s) before starting docker.

The 'Docker Daemon Config' test searches for the oldest running docker process (https://github.com/docker/docker-bench-security/blob/master/helper_lib.sh#L44) and picks up early-docker instead of docker and ends up running config tests on it. Any reason why we look for oldest running docker process instead of newest?

**Comment by dhawal55** [comment]
I have a workaround for this by replace this [line](https://github.com/docker/docker-bench-security/blob/master/helper_lib.sh#L44) with:
`for PID in $(pgrep -x -o "$PROC")`
where $PROC = "docker daemon" instead of just docker

**Comment by dhawal55** [comment]
I have a fix for this by replace [line](https://github.com/docker/docker-bench-security/blob/master/helper_lib.sh#L44) with:
`for PID in $(pgrep -f -n "$PROC")`

where $PROC is "docker daemon" instead of "docker"

**Comment by konstruktoid** [comment]
Hi @dhawal55, and thanks for reporting this and creating a PR, but it seems to be duplicate of #124 and #127.

**Comment by dhawal55** [comment]
Oh ya, thanks for pointing it out. I love that i ended up with the same solution as you did. Any reason why the fix is not merged yet?

**Comment by konstruktoid** [comment]
Well, I didn't get any response (https://github.com/docker/docker-bench-security/issues/124#issuecomment-197487316) and it hasn't been brought up since.

I'll go with your https://github.com/docker/docker-bench-security/pull/153, since there isn't any conflicts.

**Comment by konstruktoid** [comment]
#153 merged. Closing.
Thanks @dhawal55 for your contribution.
